src/qt/locale/bitcoin_zh_TW.ts

```diff
@@ -1651,8 +1651,8 @@ Signing is only possible with addresses of the type 'legacy'.</source>
         <translation>金額</translation>
     </message>
     <message>
-        <source>Enter a Bitcoin address (e.g. %1)</source>
-        <translation>輸入 比特幣地址 (比如說 %1)</translation>
+        <source>Enter a Dogecoin address (e.g. %1)</source>
+        <translation>輸入 狗狗幣地址 (比如說 %1)</translation>
```

Comment:
Remove space and retouch the sentence.
```suggestion
        <translation>輸入狗狗幣地址(例如： %1)</translation>
```